AK/Platform.h

```diff
@@ -168,16 +168,20 @@
 #endif
 
 #ifndef AK_OS_SERENITY
+#    ifdef AK_OS_WINDOWS
+// FIXME: No idea where to get this, but it's probably 4096 anyway :^)
```

Comment:
Pretty much. https://devblogs.microsoft.com/oldnewthing/20210510-00/?p=105200

Reply:
will remove the "probably"